File: engine/engine-model/src/main/scala/tools/packet/cashshop/operation/ShowWishList.scala
package tools.packet.cashshop.operation

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class ShowWishList(private var _sns: java.util.List[Integer], private var _update: Boolean) extends PacketInput {
  def sns: java.util.List[Integer] = _sns

  def update: Boolean = _update

  override def opcode(): SendOpcode = SendOpcode.CASH_SHOP_OPERATION
}

File: engine/engine-model/src/main/java/tools/packet/cashshop/operation/ShowWishList.java
package tools.packet.cashshop.operation;

import java.util.List;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record ShowWishList(List<Integer> sns, Boolean update) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.CASH_SHOP_OPERATION;
   }
}